src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActivityThreadReplies.test.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import ActivityThreadReplies from '../ActivityThreadReplies.js';
+import replies from '../../../../../__mocks__/replies';
+
+describe('src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplies', () => {
+    const defaultProps = {
```

Comment:
Minor, but it's best to inline these inside `getWrapper` so they can't inadvertently be shared/mutated between tests.